README.md

```diff
@@ -85,6 +91,19 @@ known state; however Solr will reply with an error if the collections can’t be
 
 Again, this step will take a few minutes.
 
+### Create Solr Collection Alias
```

Comment:
I think these steps, rather than documented, they should be added to the end of `docker/scxa-solrcloud-bootstrap-entrypoint.sh`. You can find the endpoints for the requests at https://lucene.apache.org/solr/guide/7_1/collections-api.html#createalias.